package.json

```diff
@@ -8,10 +8,10 @@
     "url": "https://github.com/adobe/react-spectrum"
   },
   "browserslist": [
-    "chrome >= 61",
-    "firefox >= 60",
-    "safari >= 11",
-    "edge >= 16"
+    "chrome >= 80",
```

Comment:
should we go further? i don't think we've tested on 80 in a while, Chrome is now at 98